lib/mcb/commands/providers/discard.rb

```diff
@@ -0,0 +1,14 @@
+summary "Discards a provider"
+usage "discard provider <provider code>"
+param :provider_code, transform: ->(code) { code.upcase }
+
+run do |opts, args, _cmd|
+  MCB.init_rails(opts)
+  current_cycle = RecruitmentCycle.current
+  provider = if opts&.first&.second == current_cycle.year || opts.blank?
```

Comment:
This doesn't make sense to me

Reply:
That's where the recruitment_cycle.year argument turns up from here  
``execute_discard_provider(arguments: [provider.provider_code, "-r", recruitment_cycle.year])``
There's probably a better way to extract it, but it was really late and i was tired haha